extra/update-version.js

```diff
@@ -59,3 +60,37 @@ function tagExists(version) {
     return res.stdout.toString().trim() === version;
 }
 
+function updateWiki(oldVersion, newVersion) {
```

Comment:
This seems to have been deleted in https://github.com/louislam/uptime-kuma/commit/668dffc2c501ae2af2390faa8c25dab296f2a8d7. Perhaps this was included when you fixed the merge conflicts?